news/2018-08-30-Cities-Co-Design.html

```diff
@@ -0,0 +1,169 @@
+                <span class="fl-prefsEditor-buttons">
+                    <button class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide"> Show/Hide</button>
+                    <button class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
+                </span>
+            </div>
+
+            <!-- This is the div that will contain the Preference Editor component -->
+            <div class="flc-slidingPanel-panel flc-prefsEditor-iframe"></div>
+
+            <!--
+                This div is for the sliding panel bar that shows and hides the Preference Editor controls in the desktop view.
+                A separate panel bar for desktop displays is needed to preserve the correct tab order.
+            -->
+            <div class="fl-panelBar fl-panelBar-wideScreen">
+                <span class="fl-prefsEditor-buttons">
+                    <button class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide"> Show/Hide</button>
+                    <button class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
+                </span>
+            </div>
+        </div>
+        <!-- END markup for Preference Editor -->
+
+        <div id="skip">
+            <a href="#content">Skip to Content</a>
+        </div>
+
+        <div class="floe">
+            <div class="row">
+                <div class="columns small-12 medium-9 large-9">
+                    <header class="floe-header">
+                        <h1><a rel="home" title="Home" href="../index.html" class="floe-header-logo">floe
+                            <span class="floe-header-tagline">
+                                flexible learning for open education
+                            </span></a>
+                        </h1>
+                    </header>
+                </div>
+                <div class="columns small-12 medium-3 large-3 align-self-bottom">
+                    <nav class="floe-nav">
+                        <ul>
+                            <li><a href="index.html">News</a></li>
+                            <li><a href="../resources.html">Resources</a></li>
+                        </ul>
+                    </nav>
+                </div>
+            </div>
+
+            <div class="flc-toc-tocContainer toc"> </div>
+            <article id="content" class="floe-content floe-news-item">
+                <h2> News </h2>
+
+                <!-- BEGIN markup for news item -->
+                <h3>Inclusive Cities Co-Design</h3>
+                <time class="floe-date" datetime="2018-08-30">August 30, 2018</time>
+                <p>
+                    As part of an effort to include diverse perspectives in the creation of inclusive
+                    cities, the IDRC and Floe organized embedded co-design sessions in elementary and
+                    middle schools with grade 3, 5, and 6 students. The co-design activity was developed
```

Comment:
Not sure if we have an official style guide we follow, but APA Style on writing grade numbers may be worth looking at: http://blog.apastyle.org/apastyle/2018/03/how-to-write-grade-numbers-in-apa-style.html - also suggest rephrasing to something like: "with students in Grades 3, 5 and 6".